cnf-tests/testsuites/e2esuite/ptp/ptp.go

```diff
@@ -3,157 +3,55 @@ package ptp
 	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
 	ptpv1 "github.com/openshift/ptp-operator/api/v1"
+	ptptest "github.com/openshift/ptp-operator/test/conformance/ptp"
+	"github.com/openshift/ptp-operator/test/utils"
+	k8sclient "github.com/openshift/ptp-operator/test/utils/client"
+	"github.com/openshift/ptp-operator/test/utils/testconfig"
+	l2lib "github.com/test-network-function/l2discovery-lib"
 	v1core "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"k8s.io/utils/pointer"
-)
-
-const (
-	ptpLinuxDaemonNamespace  = "openshift-ptp"
-	ptpSlaveNodeLabel        = "ptp/test-slave"
-	ptpGrandmasterNodeLabel  = "ptp/test-grandmaster"
-	ptpGrandMasterPolicyName = "test-grandmaster"
-	ptpSlavePolicyName       = "test-slave"
-	ptpDaemonsetName         = "linuxptp-daemon"
 )
 
 var isSingleNode bool
 
 var _ = Describe("[ptp]", func() {
-
+	var fullConfig testconfig.TestConfig
 	execute.BeforeAll(func() {
-		var err error
-		isSingleNode, err = nodes.IsSingleNodeCluster()
-		Expect(err).ToNot(HaveOccurred())
-
-		if !discovery.Enabled() {
-			if isSingleNode {
-				Skip("At least two nodes are required to configure ptp tests. To use an external grandmaster please use discovery mode.")
-			}
-			Clean()
-			ptpNodes, err := nodes.PtpEnabled(client.Client)
-
-			Expect(err).ToNot(HaveOccurred())
-			Expect(len(ptpNodes)).To(BeNumerically(">", 1), "need at least two nodes with ptp capable nics")
-
-			By("Labeling the grandmaster node")
-			ptpGrandMasterNode := ptpNodes[0]
-			ptpGrandMasterNode.NodeObject, err = nodes.LabelNode(ptpGrandMasterNode.NodeName, ptpGrandmasterNodeLabel, "")
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Labeling the slave node")
-			ptpSlaveNode := ptpNodes[1]
-			ptpSlaveNode.NodeObject, err = nodes.LabelNode(ptpSlaveNode.NodeName, ptpSlaveNodeLabel, "")
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Creating the policy for the grandmaster node")
-			err = createConfig(ptpGrandMasterPolicyName,
-				ptpGrandMasterNode.InterfaceList[0],
-				"-2",
-				"-a -r -r",
-				ptpGrandmasterNodeLabel,
-				pointer.Int64Ptr(5))
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Creating the policy for the slave node")
-			err = createConfig(ptpSlavePolicyName,
-				ptpSlaveNode.InterfaceList[0],
-				"-s -2",
-				"-a -r",
-				ptpSlaveNodeLabel,
-				pointer.Int64Ptr(5))
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Restart the linuxptp-daemon pods")
-			ptpPods, err := client.Client.Pods(ptpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
-			Expect(err).ToNot(HaveOccurred())
-
-			for _, pod := range ptpPods.Items {
-				err = client.Client.Pods(ptpLinuxDaemonNamespace).Delete(context.Background(), pod.Name, metav1.DeleteOptions{GracePeriodSeconds: pointer.Int64Ptr(0)})
-				Expect(err).ToNot(HaveOccurred())
-			}
+		client.Client = client.New("")
```

Comment:
@fedepaol Are you suggesting to move the entire content of this file to the ptp-operator project ? @aneeshkp  would that be ok for us ?

Reply:
Yep, these tests are probably here for legacy reasons, but if they add value to the ptp operator test suite it'd make sense to move them there. If they don't add value or they are duplicates, they can even be removed. I don't think these are integration-like tests that require multiple features to be deployed.

Reply:
If we decide to keep them here, I'd rather have some duplication than consuming an external dependency for doing the test wiring, because it would be yet another dependency that we must be careful not to break.

Reply:
I created the following PR in ptp-operator to address comments in this PR: 
https://github.com/openshift/ptp-operator/pull/252

Reply:
fixed